package.json

```diff
@@ -1,13 +1,19 @@
 {
-  "name": "@salesforce/plugin-template",
-  "description": "A template repository for sfdx plugins",
+  "name": "@salesforce/plugin-info",
+  "description": "Plugin for accessing cli info from the command line",
   "version": "1.0.0",
   "author": "Salesforce",
   "bugs": "https://github.com/forcedotcom/cli/issues",
   "dependencies": {
     "@oclif/config": "^1",
     "@salesforce/command": "^3.1.3",
     "@salesforce/core": "^2.25.1",
+    "@salesforce/kit": "^1.5.17",
+    "@salesforce/ts-types": "^1.5.20",
+    "fs-extra": "^10.0.0",
```

Comment:
fse is a sizeable dependency.  If you're already using sfdx-core, then you can use the `fs` that it includes, which has some jsonreading helpers.

Another option would be to `JSON.parse(await fs.promises.readFile('whatever')` from the native node `fs`.

Decide based on what else you might in the future.